en/docs/quick-starts/claim-config-sample.md

```diff
@@ -0,0 +1,26 @@
+# Request Attributes for the Application
+
+This page guides you through  [configuring attributes for service provider](insertlink) using the WSO2 IS 
```

Comment:
Shall we make this WSO2 Identity Server (IS) since it is the first instance of usage ?